docs/providers/fn/cli-reference/create.md

```diff
@@ -0,0 +1,84 @@
+<!--
+title: Serverless Framework Commands - Fn - Create
+menuText: create
+menuOrder: 1
+description: Creates a new Service in your current working directory
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/fn/cli-reference/create)
+<!-- DOCS-SITE-LINK:END -->
+
+# Fn - Create
+
+Creates a new Serverless service in the current working directory based on the provided template.
+
+**Create service in current working directory:**
+
+```bash
+serverless create --template fn-nodejs
+```
+
+```bash
+serverless create --template fn-go
+```
+
+**Create service in new folder:**
+
+```bash
+serverless create --template fn-nodejs --path my-service
+```
+
+```bash
+serverless create --template fn-nodejs --path my-service
```

Comment:
`fn-go` is right? Instead of `fn-nodejs`

Reply:
Both will be there initially  Went with nodejs in all the examples since from what i have seen most serverless users are using nodejs.